draft-ietf-tls-tls13.md

```diff
@@ -3699,19 +3721,18 @@ each class of traffic keys:
 
 | Record Type | Secret | Phase |
 |:------------|--------|-------|
-| 0-RTT Handshake   | early_traffic_secret | "early handshake key expansion" |
-| 0-RTT Application | early_traffic_secret | "early application data key expansion" |
-| Handshake         | handshake_traffic_secret | "handshake key expansion" |
-| Application Data  | traffic_secret_N | "application data key expansion" |
+| 0-RTT Handshake   | client_early_traffic_secret | "early handshake key expansion" |
+| 0-RTT Application | client_early_traffic_secret | "early application data key expansion" |
+| Handshake         | [sender]_handshake_traffic_secret | "handshake key expansion" |
+| Application Data  | [sender]_traffic_secret_N | "application data key expansion" |
 
-The following table indicates the purpose values for each type of key:
+The [sender] in this table denotes the sending side. The
+following table indicates the purpose values for each type of key:
 
 | Key Type         | Purpose            |
 |:-----------------|:-------------------|
-| client_write_key | "client write key" |
-| server_write_key | "server write key" |
-| client_write_iv  | "client write iv"  |
-| server_write_iv  | "server write iv"  |
+| write_key | "write key" |
+| write_iv  | "write iv"  |
```

Comment:
Nit: Are the "write" prefixes still meaningful? It seemed they were just needed to clarify "client" and "server".

(Oh hrmf. client_write_traffic_secret_N is a bit of a mouthful, but is it said anywhere "client" and "server" refer to the writer than than the reader?)